src/panoptes/pocs/tests/test_camera.py

```diff
@@ -301,7 +301,7 @@ def test_is_temperature_stable(camera):
     if camera.is_cooled_camera:
         camera.target_temperature = camera.temperature
         camera.cooling_enabled = True
-        time.sleep(1)
+        camera.wait_for_stable_temperature(blocking=True)
```

Comment:
This would make it match the line you replaced

```suggestion
        camera.wait_for_stable_temperature(blocking=True, sleep_delay=1)
```